.azure-pipelines/scripts/publish-codecov.sh

```diff
@@ -7,7 +7,8 @@ set -o pipefail -eu
 
 output_path="$1"
 
-curl --silent --show-error https://codecov.io/bash > codecov.sh
+curl --silent --show-error https://ansible-ci-files.s3.us-east-1.amazonaws.com/ansible-test/codecov.sh > codecov.sh
+curl --silent --show-error https://ansible-ci-files.s3.us-east-1.amazonaws.com/ansible-test/codecov.sh-sha256sum | shasum --check --algorithm 256
```

Comment:
I don't think this gains us anything. We trust our S3 bucket, and if we don't, we can't trust the SHA either.

```suggestion
```